tests/11-getting-started.sh

```diff
@@ -59,6 +58,12 @@ docker run --rm -i --net ${TEST_NET} -l "id.service3" --cap-add NET_ADMIN ${DEMO
 }
 
 monitor_clear
+echo "------ pinging service1 from service2 ------"
```

Comment:
I switched the order of testing the positive / negative cases since by default, we allow traffic when there are no policies added. Even though there is a policy added right before this, testing the negative case immediately after it is added ensures that policy addition / policy enforcement updates are not taking too long.